packages/expo-background-task/ios/BackgroundTaskScheduler.swift

```diff
@@ -73,6 +122,8 @@ public class BackgroundTaskScheduler {
         throw CouldNotRegisterWorkerTask("Too many pending task requests.")
       case .notPermitted:
         throw CouldNotRegisterWorkerTask("Task request not permitted.")
+      case .immediateRunIneligible:
+        throw CouldNotRegisterWorkerTask("Task request not illegible.")
```

Comment:
Should this be "not eligible" or "ineligible"?
```suggestion
        throw CouldNotRegisterWorkerTask("Task request not eligible.")
```